### ISSUE TITLE
SQS queue connector does not track job tries

### ISSUE THREAD

**Issue by georgeboot** [laravel/framework#35444]
- Laravel Version: 8.16.0
- PHP Version: 8.0.0
- Database Driver & Version: MySQL 8

### Description:
The built-in sqs queue connector does not support job tries. Laravel Vapor does support this, with a custom sqs job:

https://github.com/laravel/vapor-core/blob/5b03dd3994128684053c0c56a6c78c941fcc98f4/src/Queue/VaporJob.php#L15-L33

Vapor's SqsJob will re-create a job every time it is released, instead of updating the visibility timeout as the default SqsJob does:

https://github.com/laravel/framework/blob/09bd79531d3f958f02fa34f1f14197b1716ef29b/src/Illuminate/Queue/Jobs/SqsJob.php#L50-L592

### TL;DR

Are you willing to accept a PR to master to port this behaviour over into Laravel itself? Or are there any specific reasons not to do this?

**Comment by themsaid** [comment]
Laravel relies on `ApproximateReceiveCount` to find the number of attempts and it's working for non-serverless applications. Vapor has a special handling for this but it requires deleting the job and pushing a new one, we can't enforce this for regular apps.

You'll need to replicate Vapor's way in the bridge package you're using.
